central/notifiers/cscc/cscc.go

```diff
@@ -77,9 +77,6 @@ func newCSCC(protoNotifier *storage.Notifier, cryptoCodec cryptocodec.CryptoCode
 	decCreds := conf.ServiceAccount
 	var err error
 	if env.EncNotifierCreds.BooleanSetting() {
-		if protoNotifier.GetNotifierSecret() == "" {
```

Comment:
I think the secret is `cscc.ServiceAccount`? If so, that has to exist (see Validate). So maybe we keep that validation still?

Reply:
If `Validate` fails, this piece of code will never be reached. So removing it is ok.

Reply:
`Validate` will _not_ fail if encrypted secrets is enabled.

See the 2nd param here: https://github.com/stackrox/stackrox/blob/68aea4d8ef00806b0036d55640b2ce83a33a07f0/central/notifiers/cscc/cscc.go#L73

Which is used to skip the check in validate: https://github.com/stackrox/stackrox/blob/68aea4d8ef00806b0036d55640b2ce83a33a07f0/central/notifiers/cscc/validate.go#L26